lib/cylc/suite_host.py

```diff
@@ -147,11 +151,26 @@ def get_suite_host(self):
                 self._host_name_pref = self.get_hostname()
         return self._host_name_pref
 
+    def get_user(self):
+        """Return name of current user."""
+        return self._get_user_pwent().pw_name
```

Comment:
Oh yes. (I was looking at `remote_users`). Apparently it can be done for dictionary keys too.